src/compas_timber/fabrication/double_cut.py

```diff
@@ -102,7 +102,7 @@ def orientation(self):
 
     @orientation.setter
     def orientation(self, orientation):
-        if orientation not in [OrientationType.START, OrientationType.END]:
+        if orientation not in [OrientationType.START, OrientationType.END, None]:
```

Comment:
this was also changed in another PR (`None` removed) maybe you need to merge `main` into this one?